src/imageproxy.rs

```diff
@@ -64,6 +65,18 @@ impl Error {
     }
 }
 
+/// Errors returned by get_raw_blob
+#[derive(Error, Debug)]
+#[non_exhaustive]
```

Comment:
Interesting.  Is that basically a hint that the user needs to handle the `_` case even if they covered all of the other options?  For future compat reasons?

I think I might start using that...